integration-cli/docker_api_stats_test.go

```diff
@@ -107,9 +107,22 @@ func (s *DockerSuite) TestApiStatsNetworkStats(c *check.C) {
 	if runtime.GOOS == "windows" {
 		countParam = "-n" // Ping count parameter is -n on Windows
 	}
-	pingout, err := exec.Command("ping", contIP, countParam, strconv.Itoa(numPings)).Output()
-	pingouts := string(pingout[:])
+	pingout, err := exec.Command("ping", contIP, countParam, strconv.Itoa(numPings)).CombinedOutput()
+	if err != nil && runtime.GOOS != "windows" {
```

Comment:
I think we can safely remove `runtime.GOOS != "windows` because
the test requirements are:
```go
	testRequires(c, SameHostDaemon)
	testRequires(c, DaemonIsLinux)
```